src/Pest.php

```diff
@@ -8,3 +8,8 @@ function version(): string
 {
     return '1.0.4';
 }
+
+function testDirectory(string $file = ''): string
+{
+    return TestSuite::getInstance()->testPath . $file;
```

Comment:
It is not missing a DIRECTORY_SEPARATOR between those two?

Reply:
Looks like it, should be able to just use `/` as that is platform agnostic and cleaner.

Reply:
Don't remember running into any issues with this but will add it.

Reply:
It should have caused an issue with `testDirectory('Datasets/%s.php')`, as if you used `--test-directory=test` it would return `testDatasets/%s.php` 🤔